README.md

```diff
@@ -283,6 +283,55 @@ class CancelResponse:
     status: PaymentStatus
 ````
 
+### Refund Terminal Payment
+
+#### RefundRequest - Refund Request
+```python
+@dataclass
+class RefundRequest:
+    store_id: str
+    order_id: str
+    amount: float
+```
+
+Request parameters:
+- `store_id` - the ID of your assigned store
+- `order_id` - Unique order ID generated by Kody
```

Comment:
Are you sure this is generated by Kody? It seems to be provided by client, we generate the payment_id

Reply:
From what I see in the code, yes, during payment creation for the first time we have this: 

`orderId = request.orderId ?: UUID.randomUUID()`

Reply:
Ah, I see what you mean, now !